lib/ui/widgets/shared/custom_material_button.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:flutter/material.dart';
 
 class CustomMaterialButton extends StatelessWidget {
+  @Deprecated('Use the native FilledButton instead')
```

Comment:
Is deprecation even needed? given that this isn't a library, and all instances of CustomMaterialButton have been changed in this PR (supposedly)

Reply:
I realised that, well fair enough, I'll remove it